sjb/config/test_cases/test_branch_origin_web_console.yml

```diff
@@ -2,14 +2,40 @@
 parent: 'common/test_cases/origin_built_release.yml'
 extensions:
   actions:
+    - type: "script"
+      title: "determine binary and image version"
+      repository: "origin"
+      script: |-
+        if [[ "${PULL_BASE_REF}" = "enterprise-"* ]]; then
+          if [ "${PULL_BASE_REF}" == "enterprise-3.6" ]; then
+            echo "IMAGE_VERSION=v3.6.1" > IMAGE_VERSION_VAR
+          elif [ "${PULL_BASE_REF}" == "enterprise-3.8" ]; then
+            echo "IMAGE_VERSION=v3.9" > IMAGE_VERSION_VAR
+          else
+            echo "IMAGE_VERSION=${PULL_BASE_REF/enterprise-/v}" > IMAGE_VERSION_VAR
+          fi
+        else
+          echo "IMAGE_VERSION=latest" > IMAGE_VERSION_VAR
+        fi
+        source IMAGE_VERSION_VAR
+        sudo docker pull openshift/origin:"${IMAGE_VERSION}"
+        sudo docker create --name temp-container openshift/origin:"${IMAGE_VERSION}"
+        sudo docker cp temp-container:/usr/bin/oc ./
+        sudo docker rm temp-container
+        sudo mv -f ./oc /bin/oc
```

Comment:
I'd probably move it to a tmp folder instead of overwriting /bin/oc, then use the full path when invoking it

Reply:
Or leave it where it is and call it with `./oc`

Reply:
Agreed

Reply:
The thing is that the hack/test-headless.sh test-integration is calling the oc binary, if we wont put it into PATH the job will fail

Reply:
ACK